### DISCUSSION TITLE
Ability to open camera roll in external gallery

### DISCUSSION THREAD

**Discussion by warmx3** [rodit/SnapMod#203] [Ideas & Feature Suggestions]
The built in snap chat "camera roll" is a mess to navigate through and impossible to find something for those with large galleries. Would it be possible for it to open the android gallery so that pictures can be easily found? The ability to sort through albums would make the whole experience much more seamless.

**Comment by rodit** [comment]
Just share from your favourite gallery to Snapchat

**Reply by warmx3** [reply]
What do you mean? is it possible to open an external gallery app through snapchat?
